core/templates/packages.el.template

```diff
@@ -1,31 +1,63 @@
-;;; packages.el --- %LAYERNAME% Layer packages File for Spacemacs
+;;; packages.el --- %LAYER_NAME% layer packages file for Spacemacs.  -*- lexical-binding: t; -*-
```

Comment:
Is there a reason to force lexical-binding by default? I'm not against it, just to know.